torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -121,6 +136,37 @@ std::shared_ptr<FutureMessage> py_rpc_python_udf(
                             MessageType::PYTHON_CALL));
 }
 
+PyRRef py_remote_python_udf(
+    RpcAgent& agent,
+    const WorkerId& dst,
+    const std::string& pickledPythonUDF) {
+
+  auto& ctx = RRefContext::getInstance();
+  if (ctx->getWorkerId() == dst.id_) {
+    auto ownerRRef = ctx->createOwnerRRef<py::object>(dst.id_);
+    agent.send(
```

Comment:
We shouldn't throw away the Future returned by send. User should have a way of being notified of exceptions that happen in the context of this RPC. Even temporarily, we should at least print an error message for this.